content/concept-accounts.md

```diff
@@ -0,0 +1,123 @@
+# RCL Accounts
+
+An "Account" in the Ripple Consensus Ledger represents a holder of XRP and a sender of transactions. The core elements of an account are:
+
+- An identifying **address**, such as `rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn`
+- An account can send [transactions](reference-transaction-format.html) to modify the Ripple Consensus Ledger, for a variety of purposes including:
+    - Sending XRP to a recipient, immediately or through locking mechanisms like payment channels and escrow.
+    - Issuing, redeeming, or sending non-XRP assets and currencies.
+    - Trading in the decentralized exchange.
+    - Changing settings such as authorization settings, balance limits, or the behavior of assets issued by the account.
+- An **XRP balance**. Some of this XRP is set aside for the [Reserve](concept-reserves.html).
+- A **sequence number**, starting at 1 and increasing with each transaction sent from this account. No transaction can be included in a ledger unless the transaction's sequence number matches its sender's next sequence number.
+- A **history of transactions** that affected this account and its balances.
+
+In the ledger's data tree, an account's core data is stored in the [AccountRoot](reference-ledger-format.html#accountroot) ledger node type. An account can also be the owner (or partial owner) of several other types of data.
+
+**Tip:** An "Account" in the Ripple Consensus Ledger is somewhere between the financial usage (like "bank account") and the computing usage (like "UNIX account"). Non-XRP currencies and assets aren't stored in an RCL Account itself; each such asset is stored in an accounting relationship called a "Trust Line" that connects two parties.
+
+
+## Addresses
+
+{% include 'data_types/address.md' %}
+
+The conversion from a public key to an address involves a one-way hash function, so it is possible to confirm that a public key matches an address but it is impossible to derive the public key from the address alone. (This is part of the reason why signed transactions include the public key _and_ the address of the sender.) For more technical details, see [Address Encoding](#address-encoding).
+
+The process of "creating" an address is the purely mathematical task of generating a key pair and calculating the address from it, so an address can exist separately from its presence as an account object in the ledger. To actually make an account for an address, someone must send the address enough XRP to satisfy the reserve. This is called _funding_ the account. Whether or not the address is a funded account, you can use the address (and the key pair it's derived from) as a [regular key](reference-transaction-format.html#setregularkey) or as part of a [signer list](reference-transaction-format.html#multi-signing).
```

Comment:
I restructured this section a little to clear up some of the confusion. After agonizing for quite a while trying to come up with a wording that made more sense to me than "purely mathematical task," I threw in the towel and settled for "Creating a valid address is a strictly mathematical task starting with a key pair." My thought process is:

- The condescending feel may have been coming in part from some other pieces of the passage, such as the scare quotes around "creating" and the the word "actually" splitting the infinitive in the following sentence. By removing those, I hope the adverb is less likely to offend.
- I changed to "strictly" because the strictness sounds more academic than the ideological connotation of "purely."
- I felt it was still important to retain an adverb to rule out any other bits of non-math that people could unknowingly assume are involved in creating an address.

I also moved the more important bits about funding accounts and using unfunded accounts first. I tried to include a paragraph explaining that anyone can fund an account but only the holder of the secret key can control the account, but ultimately cut it in the name of simplicity. The resulting passage looks like this:

> Any valid address can become an account in the Ripple Consensus Ledger by receiving a [Payment][] of XRP, as long as the amount of XRP delivered is greater than or equal to the [account reserve](concept-reserves.html). This is called _funding_ the account. You can also use an address that has not been funded to represent a [regular key](reference-transaction-format.html#setregularkey) or a member of a [signer list](reference-transaction-format.html#multi-signing). Only a funded account can be the sender of a transaction.
>
> Creating a valid address is a strictly mathematical task starting with a key pair. You can generate a key pair and calculate its address entirely offline without communicating to the Ripple Consensus Ledger or any other party. The conversion from a public key to an address involves a one-way hash function, so it is possible to confirm that a public key matches an address but it is impossible to derive the public key from the address alone. (This is part of the reason why signed transactions include the public key _and_ the address of the sender.)
> 
> For more technical details of how to calculate a Ripple address, see [Address Encoding](#address-encoding).